gammapy/estimators/points/core.py

```diff
@@ -850,3 +873,79 @@ def resample_axis(self, axis_new):
             fluxpoints.append(fp_new)
 
         return self.__class__.from_stack(fluxpoints, axis=axis_new)
+
+
+class LightCurve(FluxPoints):
+    """Specialized 'FluxPoints' heir class for lightcurves"""
+
+    @classmethod
+    def read(cls, filename, **kwargs):
+        kwargs.setdefault("format", "lightcurve")
+        return super().read(filename, **kwargs)
+
+    @classmethod
+    def from_table(cls, table, **kwargs):
+        table = table_standardise_units_copy(table)
+
+        if super()._table_guess_format(table) != "lightcurve":
+            raise ValueError("The input table is not a lightcurve!")
+
+        kwargs.setdefault("format", "lightcurve")
+
+        return super().from_table(table, **kwargs)
+
+    def to_stingray_lightcurve(self):
+        """Transform to a 'stingray.Lightcurve' object.
+        Only objects with a single energy bin are allowed.
+        """
+
+        if self.geom.axes["energy"].nbin > 1:
+            raise ValueError("Only lightcurves with 1 energy bin are supported.")
+
+        try:
+            from stingray import Lightcurve
+
+            counts = self.counts.data
```

Comment:
I am still uncomfortable with exporting just the counts as a reliable unit.
Since the `Lighcurve` object contains a spectral model, maybe we can scale the flux by the model and use some "scaled counts"?

Reply:
This is a good idea, I should probably investigate stingray to see what would be the best alternative here.